leetcode/src/231.c

```diff
@@ -1,7 +1,6 @@
-bool isPowerOfTwo(int n)
-{
-    if (!n)
-        return false;
-    while (n % 2 == 0) n /= 2;
-    return n == 1;
-}
\ No newline at end of file
+// Without loops/recursion.
+// Runtime: O(1)
+// Space: O(1)
+bool isPowerOfTwo(int n){
+    return (n > 0) && ((n & (n - 1)) == 0);

```

Comment:
Shouldn't this be `&&`?
```suggestion
    return (n > 0) && ((n && (n - 1)) == 0);
```

Reply:
No, actually this is a bitwise and for 2 integers https://en.wikipedia.org/wiki/Bitwise_operations_in_C#Bitwise_AND_& .